releasenotes/notes/minor-mcx-improvements-a1ed81f83275ce35.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features_synthesis:
+  - |
+    The synthesis function :func:`.synth_mcx_1_clean_b95` now produces a circuit
+    with fewer CX-gates.
```

Comment:
can you please update the tests in 
https://github.com/Qiskit/qiskit/blob/main/test/python/synthesis/test_multi_controlled_synthesis.py ?

Reply:
Done in [89c48eb](https://github.com/Qiskit/qiskit/pull/14093/commits/89c48eb808ba7a26fff10ceeead06aeb2c9d4fb0)